packages/storybook/stories/components/AlphabetNavigation.stories.tsx

```diff
@@ -0,0 +1,432 @@
+    },
+  },
+};
+
+export const ActiveLinkCurrentLetter: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'link',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+    currentLetter: 'A',
+  },
+  parameters: {
+    pseudo: { active: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const HoverLink: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'link',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { hover: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const HoverLinkCurrentLetter: Story = {
+  args: {
+    currentLetter: 'A',
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'link',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { hover: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const FocusButton: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'button',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { focus: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const FocusVisibleButton: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'button',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { focusVisible: true, focus: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const FocusButtonCurrentLetter: Story = {
+  args: {
+    currentLetter: 'A',
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'button',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { focus: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const ActiveButton: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'button',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { active: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const ActiveButtonCurrentLetter: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'button',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+    currentLetter: 'A',
+  },
+  parameters: {
+    pseudo: { active: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const HoverButton: Story = {
+  args: {
+    alphabet: [{ letter: 'A', availability: true }],
+    component: 'button',
+    handleLetterClick: (selectedLetter: string) => selectedLetter,
+  },
+  parameters: {
+    pseudo: { hover: true },
+    status: {
+      type: 'ALPHA',
+    },
+  },
+};
+
+export const HoverButtonCurrentLetter: Story = {
```

Comment:
The component serves two distinct purposes:

- Link: When the intention is to navigate to different pages, such as the example /nl/products/alphabet/a, we use the 'Link' designation. In this case, the component directs users to various pages for navigation.

- Button: After users have arrived at a specific destination, such as /nl/products/alphabet/a, the component changes its behavior. Instead of using links, it renders buttons. These buttons are responsible for fetching data based on availability.